### PR TITLE
Second batch of PR review comments addressed

### PR DESCRIPTION
**Pull request by ajcasagrande**
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595519578
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595523551
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595524138
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595525260
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595572991
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595575088
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#pullrequestreview-613781892
https://github.com/edgexfoundry-holding/rfid-llrp-inventory-service/pull/18#discussion_r595588978

Signed-off-by: Anthony Casagrande <anthony.j.casagrande@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Are there any new imports or modules? If so, what are they used for and why?
No

## Are there any specific instructions or things that should be known prior to reviewing?
No

## Other information
N/A

### CODE REVIEW

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ## PR Checklist
 Please check if your PR fulfills the following requirements:
 
-- [ ] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
 - [ ] Tests for the changes have been added (for bug fixes / features)
 - [ ] Docs have been added / updated (for bug fixes / features)
 
+<!-- If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/.github/CONTRIBUTING.md -->
```
[the remark this answers is not in the document]
Comment (ajcasagrande) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: ah, ok

File: internal/llrp/util_test.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
+//
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package llrp
```
[the remark this answers is not in the document]
Comment (ajcasagrande) [reply internal/llrp/util_test.go:6]: They never existed in the first place. This was my way of noting that there probably should be some.

File: internal/logutil/logwrap.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+// SPDX-License-Identifier: Apache-2.0
+
+package logutil
+
+import (
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	"os"
+)
+
+type LogWrap struct {
```
[the remark this answers is not in the document]
Comment (lenny-intel) [reply internal/logutil/logwrap.go:13]: Yep, agree.

File: internal/llrp/util_test.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (lenny-intel) [reply internal/llrp/util_test.go:6]: So are unit tests planned for another PR? i.e. we can merge this one?

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**